SConstruct

```diff
@@ -2888,6 +2888,23 @@ BOOST_AUTO_TEST_CASE(f)
 		if register_runtime_test_link_targets:
 			self.check_boost_test(context)
 
+	# dylibbundler is used to embed libraries into macOS app bundles,
+	# however it's only available on macOS.  While required for macOS
+	# builds, builds should not fail on other operating systems if
+	# it's absent.
+	@_custom_test
+	def _check_dylibbundler(self,context):
+		context.Display('%s: checking whether dylibbundler is installed...' % self.msgprefix)
+		if sys.platform != 'darwin':
+			context.Result('n/a')
+			return True
```

Comment:
Tests are not required to return any particular value, and many use a bare `return`.  Unacceptable results are reported via exception, rather than return code.

Reply:
Fair enough.  I'll change these to just return statements, then.